dist/rpm/apptainer.spec.in

```diff
@@ -132,27 +132,27 @@ mkdir -p $RPM_BUILD_ROOT%{_mandir}/man1
 make -C builddir DESTDIR=$RPM_BUILD_ROOT install man
 
 %files
-%attr(4755, root, root) %{_libexecdir}/singularity/bin/starter-suid
-%{_bindir}/singularity
+%attr(4755, root, root) %{_libexecdir}/apptainer/bin/starter-suid
+%{_bindir}/apptainer
```

Comment:
Need a %{_bindir}/singularity as well as %{_bindir}/apptainer.

Reply:
of course you are correct, and these can be resolved by later Issue/PR rather than blocking this one

This is not a case where Athena needs to leap from Jupiter's head fully formed

Reply:
true -- however, please make this a generic issue to maintain backward compatibility containers ... amending the commit destroys the comment reference above.   Also, for the generic issue, we need to update the E2E tests to detect and enforce a backward compatibility rule.   Such E2E update does not belong in this PR

Reply:
deferred to issue #39